tests/test_commands.py

```diff
@@ -108,6 +116,22 @@ def test_change_status_specify_valid_status_name(self):
         self.assertEqual(result.output, 'Fetching possible transitions...\nOkay, the status for GOJI-311 is now "Done".\n')
 
 
+class SearchCommandTests(unittest.TestCase):
+    def test_search(self):
+        runner = CliRunner()
+        args = ['--base-url=https://example.com', 'search', 'PROJECT=GOJI']
+        result = runner.invoke(cli, args, obj=TestClient())
+        self.assertIsNone(result.exception)
```

Comment:
Should probably have a case where this exception is not None. What happens if the JQL is invalid, for example?